content/rs/references/client_references/client_ioredis.md

```diff
@@ -0,0 +1,110 @@
+---
+Title: Using Redis with Node.js (ioredis)
+description:
+weight:
+alwaysopen: false
+categories: ["RS"]
+---
+In order to use Redis with Node.js, you need to install a Node.js Redis client.  In the following sections, we demonstrate the use of [ioredis](https://github.com/luin/ioredis), a community recommended Redis client for Node.js with support for Promises built in.  The other community recommended client for Node.js developers is [node_redis]({{< relref "./client_nodejs" >}}).
```

Comment:
We try put each sentence or sentence segment on it's own line.

Reply:
We use single spaces between sentences but if you put the new sentence on a new line then you don't have to put in the space.